src/chunkserver/chunkserver-common/hdd_utils.h

```diff
@@ -5,6 +5,10 @@
 #include "chunkserver-common/chunk_interface.h"
 #include "common/chunk_with_version_and_type.h"
 #include "protocol/chunks_with_type.h"
+#include "common/event_loop.h"
```

Comment:
Not needed here. Please move it to the .cc file.
Also, please add the common/platform.h to both files (.h and .cc) at the beginning.